setup/tester.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+current_dir=$PWD
+parentdir="$(dirname "$(pwd)")"
+
+foldername="$(basename "$PWD")" 
+echo $DIR
+echo $parentdir
+echo $current_dir
+if [[ -f "./ex01/ft_print_alphabet.c" || "${foldername,,}" == "c00"  ]]; then 
```

Comment:
@hubble42 I get the following error here when I run 42tester on a Mac in a project repo:
`zsh: bad substitution`
Don't have Linux at the moment. Is this just a Mac related issue?

Reply:
yes mac related, in zsh it must be ${foldername:l}  instead  ${foldername,,} in bash, to convert the current foldername to lowercase